### DISCUSSION TITLE
[Bug] Checkboxes vanish when editing a post

### DISCUSSION THREAD

**Discussion by LekoArts** [community/community#2804] [Discussions]
## Description

When editing a discussion entry and adding checkboxes, they vanish after either clicking on the preview tab or saving the post

![2020_11_23_4fd0c6be](https://user-images.githubusercontent.com/16143594/99941760-dd74bd80-2d6e-11eb-8e5a-9fba4141cde4.gif)

**Comment by becca** [comment]
👋 hey @LekoArts, I was not able to reproduce this. Are you still seeing this happen consistently on your end?

**Reply by LekoArts** [reply]
Yep. I realized that I have "Refined GitHub" activated, but this also occurs in an incognito tab without any plugins (I use latest Chrome). Happy to provide any additional info you might need.

**Reply by becca** [reply]
Thanks for checking that! I believe this may be OS related - I'll continue digging to determine an ETA for a fix. 

Is this also happening for you in Issues and PRs, by the way?

**Reply by LekoArts** [reply]
No, this is only happening when editing an existing discussion. Editing/creating new issues or creating new discussions is working fine :)

**Reply by becca** [reply]
I take back my previous statement. I was just able to repro. Going to look into this and get back to you when it's resolved. Thank you for bringing this to my attention!
